core/server/influx-metrics.js

```diff
@@ -0,0 +1,68 @@
+'use strict'
+
+const { promisify } = require('util')
+const { post } = require('request')
+const { promClientJsonToInfluxV2 } = require('./metrics/format-converters')
+const log = require('./log')
+
+module.exports = class InfluxMetrics {
+  constructor(metricInstance, instanceMetadata, config) {
+    this._metricInstance = metricInstance
+    this._instanceMetadata = instanceMetadata
+    this._config = config
+  }
+
+  async sendMetrics() {
+    const auth = {
+      user: this._config.username,
+      pass: this._config.password,
+    }
+    const request = {
+      uri: this._config.url,
+      headers: { 'Content-Type': 'application/x-www-form-urlencoded' },
+      body: this.metrics(),
+      timeout: this._config.timeoutMillseconds,
+      auth,
+    }
+    const postAsync = promisify(post)
```

Comment:
This could be moved outside of the class so it's not recreated for each call.

Reply:
👍  done